docs/docs/architecture/orchestration.md

```diff
@@ -51,7 +51,7 @@ Post-installation the CLI provides a configuration for [accessing the cluster us
 The `kubeconfig` file provides the credentials and configuration for connecting and authenticating to the API server.
 Once configured, orchestrate the Kubernetes cluster via `kubectl`.
 
-Make sure to keep the state files such as `terraform.tfstate` in the workspace directory to be able to manage your cluster later on.
+Make sure to keep the Terraform subdirectory such as `constellation-terraform` in the workspace directory to be able to manage your cluster later on.
 Without it, you won't be able to modify or terminate your cluster.
```

Comment:
```suggestion
```
I'm not sure if this hint is necessary at all after explaining what the workspace directory is. What do you think?